include/secp256k1_generator.h

```diff
@@ -24,6 +24,12 @@ typedef struct {
     unsigned char data[33];
 } secp256k1_generator;
 
+/** Standard secp256k1 generator G */
+extern const secp256k1_generator secp256k1_generator_const_g;
+
+/** Alternate secp256k1 generator from Elements Alpha */
```

Comment:
We really don't need to support Elements Alpha.

Reply:
It's useful to have a standard second generator though for non-CA applications.